src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -119,30 +160,30 @@ public void findPersonInTransaction(Name name) throws PersonNotFoundException {
     }
 
     @Override
-    public Set<Transaction> findTransactionsWithPayer(Person person) throws TransactionNotFoundException {
+    public boolean findTransactionsWithPayer(Person person) throws PersonFoundException {
         Set<Transaction> matchingTransactions = addressBook.getTransactionList()
                 .stream()
                 .filter(transaction -> transaction.getPayer().equals(person))
                 .collect(Collectors.toSet());
 
-        if (!matchingTransactions.isEmpty()) {
-            return matchingTransactions;
+        if (matchingTransactions.isEmpty()) {
+            return true;
```

Comment:
This is really misleading !!! your findTransactionsWithPayer returns true when it doesn't find any matching transactions. Not really appropriate name.